salt/scripts.py

```diff
@@ -14,6 +14,8 @@
 from random import randint
 
 # Import salt libs
+from salt import cloud, defaults
```

Comment:
Running `PYTHONPATH=/path/to/dev/ /path/to/virtualenv/bin/python /path/to/dev/salt/salt-cloud` would constantly crash because it wouldn't find the modules.